### PR TITLE
Add description about CRL support for configuring the Ingress Gateway…

### PR DESCRIPTION
**Pull request by dseapy**
… using SDS

---
name: Pull Request
about: Adds documentation for an enhancement.
---

Please provide a description for what this PR is for.

Adds documentation for how to provide a CRL when configuring mtls on the Ingress Gateway using SDS.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[X] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastrcture

### CODE REVIEW

**Comment by dseapy** [comment]
Didn't make changes to `content_zh`, though I noticed there is a another version of this file in there.  I'll take a quick look in the README's again to see how that is done.

File: content/docs/tasks/traffic-management/ingress/secure-ingress-sds/index.md [hunk 1]
```diff
@@ -462,6 +462,10 @@ the credentials of the ingress gateway by deleting its secret and creating a new
 The server uses the CA certificate to verify
 its clients, and we must use the name `cacert` to hold the CA certificate.
 
+{{< tip >}}
```
Comment (robinpercy) [inline content/docs/tasks/traffic-management/ingress/secure-ingress-sds/index.md]: @dseapy, I just came across your PRs for this feature as I was doing my own investigation into CRL support. I think the feature is exactly what I need, but I can't really tell how to use it based on these docs. A more complete example would be very helpful.

Comment (dseapy) [reply content/docs/tasks/traffic-management/ingress/secure-ingress-sds/index.md]: I'm happy to add more, I mainly didn't want to distract or complicate things for those reading this guide without needing a CRL.  So I tried to match how line 463 says "use name `cacert`" which matches the original line 470.

So you just need to add:
```
--from-file=crl=/some/path/to/your-crl.pem
```
so the full example would be:
```
kubectl create -n istio-system secret generic httpbin-credential  \
--from-file=key=httpbin.example.com/3_application/private/httpbin.example.com.key.pem \
--from-file=cert=httpbin.example.com/3_application/certs/httpbin.example.com.cert.pem \
--from-file=cacert=httpbin.example.com/2_intermediate/certs/ca-chain.cert.pem \
--from-file=crl=/some/path/to/your-crl.pem
```

**Comment by robinpercy** [comment]
Oh got it. Thanks for the example!

**Pull request closed without merging**